.github/workflows/pjsip-2.8.yml

```
@@ -0,0 +1,94 @@
+name: pjsip CI
+
+on:
+  pull_request:
+    paths:
+      - 'recipes/pjsip/2.8/**'
+      - 'build.py'
+      - '.github/workflows/pjsip-2.8.yml'
+  push:
+    branches:
+      - trassir-ci
+    paths:
+      - 'recipes/pjsip/2.8/**'
+      - 'build.py'
+      - '.github/workflows/pjsip-2.8.yml'
+
+env:
+  CONAN_PASSWORD: ${{ secrets.BintrayApiKey }}
+  CONAN_REFERENCE: 'pjsip/2.8'
+
```

pjsip вроде бы сишный, тут стоит указать
```suggestion
  IS_PURE_C: true

```